src/io/aleph/dirigiste/Pool.java

```diff
@@ -293,9 +296,16 @@ private void startControlLoop(int duration, int iterations) {
             while (!_isShutdown) {
 
                 iteration = (iteration + 1) % iterations;
+                boolean isControlPeriod = iteration == 0;
 
                 long start = System.currentTimeMillis();
 
+                // During the control period, we need to ensure we have an exclusive access to the queues to ensure
+                // we compute a stable utilization and prevent objects creation.
```

Comment:
If we compute utilization incorrectly outside of control periods, won't we still run into problems, since we look at the 99 pct, which may not correspond to the stats from the control period iteration?

Reply:
480879cc20a072c5c6be80520a9fe13a66ec7e8d